be/src/olap/delta_writer.h

```diff
@@ -124,6 +124,10 @@ class DeltaWriter {
 
     int64_t total_received_rows() const { return _total_received_rows; }
 
+    int64_t DeltaWriter::num_rows_filtered() {
```

Comment:
This does not compile, please move it to `.cpp` file.

In header, it should just be 

```cpp
int64_t num_rows_filtered() const;
```

In cpp file (add the const)

```cpp
int64_t DeltaWriter::num_rows_filtered() const {
    return _rowset_writer == nullptr ? 0 : _rowset_writer->num_rows_filtered();
}
```